modules/gsp-cluster/main.tf

```diff
@@ -196,7 +196,7 @@ module "kiam-system" {
   enabled        = 1
   namespace      = "kiam-system"
   chart_git      = "https://github.com/alphagov/gsp-kiam-system"
```

Comment:
Not a problem, not even part of this PR :D but this lack of `.git` suffix is bothering me for lack of consistency :D